client/landing/jetpack-cloud/components/scan-history-item/index.jsx

```diff
@@ -25,61 +28,216 @@ class ScanHistoryItem extends Component {
+
+			case 'database':
+				return null;
+
+			case 'none':
+			default:
+				return translate( 'Miscellaneous vulnerability' );
+		}
+	}
+
+	getTitle( threat ) {
+		// This should be temprary since this data should be coming from the api
+		// and not something that we should change to accompadate the results.
+		const { filename, extension } = threat;
+
+		const basename = s => s.replace( /.*\//, '' );
+
+		switch ( this.getDetailType( threat ) ) {
+			case 'core':
+				return translate( 'Infected core file: {{filename/}} ', {
+					components: {
+						filename: <code className="threat-item__alert-filename">{ basename( filename ) }</code>,
+					},
+				} );
+
+			case 'file':
+				return translate( 'The file {{filename/}} contains a malicious code pattern.', {
+					components: {
+						filename: <code className="threat-item__alert-filename">{ basename( filename ) }</code>,
+					},
+				} );
+
+			case 'plugin':
+				return translate( 'Vulnerable Plugin: {{pluginSlug/}} (version {{version/}})', {
+					components: {
+						pluginSlug: <span className="threat-item__alert-slug">{ extension.slug }</span>,
+						version: <span className="threat-item__alert-version">{ extension.version }</span>,
+					},
+				} );
+
+			case 'theme':
+				return translate( 'Vulnerable Theme {{themeSlug/}} (version {{version/}})', {
+					components: {
+						themeSlug: <span className="threat-item__alert-slug">{ extension.slug }</span>,
+						version: <span className="threat-item__alert-version">{ extension.version }</span>,
+					},
+				} );
+
+			case 'database':
+				return translate( 'Database %(threatCount)d threat', 'Database %(threatCount)d threats', {
+					count: Object.keys( threat.rows ).length,
+					args: {
+						threatCount: Object.keys( threat.rows ).length,
+					},
+				} );
+
+			case 'none':
+			default:
+				return translate( 'Threat found' );
+		}
+	}
+
+	getThreatFix( threat ) {
+		switch ( this.getDetailType( threat ) ) {
+			case 'database':
+				return <>TODO:FIX THIS CASE></>;
+			default:
+				return (
+					<>
+						<p className="threat-description__section-text">{ threat.description }</p>
+						{ threat.filename ? (
+							<>
+								<p>
+									{ translate( 'Threat {{threatSignature/}} found in file:', {
+										comment:
+											'filename follows in separate line; e.g. "PHP.Injection.5 in: `post.php`"',
+										components: {
+											threatSignature: (
+												<span className="threat-description__alert-signature">
+													{ threat.signature }
+												</span>
+											),
+										},
+									} ) }
+								</p>
+								<pre className="threat-description__alert-filename">{ threat.filename }</pre>
+							</>
+						) : (
+							<p className="threat-description__section-text">{ threat.signature }</p>
+						) }
+						{ threat.context && <MarkedLines context={ threat.context } /> }
+						{ threat.diff && <DiffViewer diff={ threat.diff } /> }
```

Comment:
I think we should add a `margin-bottom` of 16 or 24px to the `DiffViewer` component. It seems to be too close to the title that is just below it.
![image](https://user-images.githubusercontent.com/3418513/79593292-a1c79600-80b1-11ea-9290-4aea23517e6f.png)